src/api/action/getActivities.ts

```diff
@@ -109,16 +115,16 @@ export async function getActivitiesAction(
     // Filter the activity refs by any provided type filters
     if (typeFilters.length > 0) {
         activityRefs = activityRefs.filter(ref => {
-            const activity = activityMeta.get(ref)!;
+            const meta = activityMeta.get(ref)!;
```

Comment:
Renamed this `meta` to avoid confusion with thinking the thing returned from `activityMeta` is the `activity`